docs/topics/time.md

```diff
@@ -0,0 +1,374 @@
+enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can also be zero, negative or infinite.
+
+### Create duration
+
+To create a `Duration`, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
+available for `Int`, `Long`, and `Double` types: `nanoseconds`, `microseconds`, `milliseconds`, `seconds`, `minutes`, 
+`hours`, and `days`.
+
+> Days refer to periods of 24 hours. They are not calendar days.
+> 
+{type = "tip"}
+
+For example:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.Duration.Companion.minutes
+
+fun main() {
+    //sampleStart
+    val fiveHundredMilliseconds: Duration = 500.milliseconds
+    val fiveSeconds: Duration = 5.seconds
+    val tenMinutes: Duration = 10.minutes
+
+    println(fiveHundredMilliseconds) // 500ms
+    println(fiveSeconds)             // 5s
+    println(tenMinutes)              // 10m
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+You can also perform basic arithmetic with `Durations`:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    val thirtySeconds: Duration = 30.seconds
+
+    println(fiveSeconds + thirtySeconds)
+    // 35s
+    println(thirtySeconds -  fiveSeconds)
+    // 25s
+    println(fiveSeconds * 2)
+    // 10s
+    println(thirtySeconds / 2)
+    // 15s
+    println(thirtySeconds / fiveSeconds)
+    // 6.0
+    println(-thirtySeconds)
+    // -30s
+    println((-thirtySeconds).absoluteValue)
+    // 30s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+### Get string representation
+
+It can be useful to have a string representation of a `Duration` so that you can print, serialize, transfer, or store it.
+
+To get a string representation, use the `.toString()` function. By default, the time is reported using each unit that is
+present. For example: `1h 0m 45.677s` or `-(6d 5h 5m 28.284s)`
+
+To configure the output, use the `.toString()` function with your desired `Duration.Unit` and number of decimal places
+as function parameters:
+
+```kotlin
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.DurationUnit
+
+fun main() {
+    //sampleStart
+    // Print in seconds with 2 decimal places
+    println(5887.milliseconds.toString(DurationUnit.SECONDS, 2))
+    // 5.89s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
+
+To print the `Duration` as an [ISO-8601-compatible](https://en.wikipedia.org/wiki/ISO_8601) string, use the [`toIsoString()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/to-iso-string.html)
```

Comment:
Here again printing is only one of use cases of getting a string representations.